examples/wheel/lib/BUILD.bazel

```diff
@@ -21,12 +21,31 @@ licenses(["notice"])  # Apache 2.0
 py_library(
     name = "simple_module",
     srcs = ["simple_module.py"],
+    imports = ["../../.."],
 )
 
 py_library(
     name = "module_with_data",
     srcs = ["module_with_data.py"],
     data = [":data.txt"],
+    imports = ["../../.."],
+)
+
+py_library(
+    name = "module_with_pip_deps",
+    srcs = ["module_with_pip_deps.py"],
+    imports = ["../../.."],
+    target_compatible_with = select({
```

Comment:
Same comment here as the other TCW usage.

Reply:
Addressed in dbb0710.